RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1771,6 +1771,31 @@ boolean acquireMP(int goal, int meat_reserve, boolean useFreeRests)
 			}
 		}
 	}
+	if (canUseSweatpants() && getSweat() >= 95) {
+		int MPtoRestore = my_mp() - goal;
+		int casts = MPtoRestore / 50;
+		casts = max(casts, (getSweat() - 90) / 5);
+		if (casts > 0) {
+						int excessMP = my_mp() + 15*casts - my_maxmp();	//if some of the restored MP would be wasted over max
```

Comment:
50*casts. Also github shows there are extra tabs. Don't know if that is also in the source code